rules/aar_import/impl.bzl

```diff
@@ -592,7 +592,6 @@ def impl(ctx):
     # Bazel developers so that `bazel build java/com/my_aar_import` will fail if the resource
     # processing or JAR merging steps fail.
     files_to_build = []
-    files_to_build.extend(resources_ctx.validation_results)  # TODO(djwhang): This should be validation.
```

Comment:
Should these be added to validation_outputs below, or are these already included? I looked through the code and I'm not sure that they're already there, but maybe I missed it